eventfiles/class_tests/event_file_views_tests.py

```diff
@@ -37,13 +54,75 @@ def test_render_files_template(self, client, sign_in):
         "event doesnt exist",
         "user not a participant in the event",
     ])
-    def test_invalid_event_id_in_file_url(self, client, sign_in, url_path, status_code):
+    def test_invalid_event_id_in_file_url(self, client, sign_in_with_creator, url_path, status_code):
         response = client.get(url_path)
         assert response.status_code == status_code
 
-    def test_unauthorized_user_redirected_to_main_page(self, client):
+    def test_unauthorized_user_redirected_to_login_page(self, client):
         response = client.get(f'{FILE_CREATION_URL}/1')
         assert response.status_code == 302
         response = client.get(response.url)
         assert response.status_code == 200
-        assertTemplateUsed(response, 'main/index.html')
+        assertTemplateUsed(response, 'login/register_login.html')
+
+    def test_all_event_files_are_shown_and_downloadable(self, sign_in_with_user2, client):
+        char_content = self.get_char_content_from_eventfile_html(client)
+        all_file_in_event1 = EventFile.objects.get_files_by_event(EVENT_ID)
+        for file in all_file_in_event1:
+            assert f'{file}' in char_content
+            assert f'{DOWNLOAD_FILE_FROM_EVENT_URL}{file}' in char_content
+            response = client.get('/file/event_file/download/', {'event': EVENT_ID, 'file_name': f'{file}'})
```

Comment:
Can you set the URL as a constant.
This one: '/file/event_file/download/'

Reply:
Done